src/main/kotlin/io/github/seggan/sf4k/extensions/SlimefunItemUtils.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.seggan.sf4k.extensions
+
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem
+import org.bukkit.inventory.ItemStack
+
+fun ItemStack.isSlimefun() = SlimefunItem.getByItem(this) != null
+fun ItemStack.asSlimefun() = SlimefunItem.getByItem(this)
+
+inline fun <reified T : SlimefunItem> ItemStack.asSlimefun(): T {
```

Comment:
`asSlimefun` implies the function returns a `SlimefunItemStack`, maybe rename it to something like `getSlimefun` or `asSlimefunItem`?